configure.ac

```diff
@@ -200,7 +200,7 @@ AC_PATH_PROG(PERL, perl)
 #   -Wall                        CC, CXX
 #
 
-PROSPECTIVE_CFLAGS="-Wall"
+PROSPECTIVE_CFLAGS="-Werror"
```

Comment:
No concerns - I'm proposing stricter checks here by having both `-Wall` and `-Werror`.